### ISSUE TITLE
Voorhees Attribution

### ISSUE THREAD

**Issue by danmcclain** [DockYard/json_api_assert#23]
I think it would only be fair to attribute Voorhees for the original idea, especially when claiming that this is a rewrite of the Voorhees library, a library that was originally contributed to by the author until I left DockYard.

![main window 2016-08-29 09-33-06](https://cloud.githubusercontent.com/assets/144138/18053030/84dc6cf2-6dcb-11e6-9872-650bfa39d1ba.png)

**Comment by bcardarella** [comment]
I'm happy to attribute if you can provide examples of where the attributions come from. I believe that this is 100% original work based off of a pipe-based method that I proposed for Voorhees https://github.com/danmcclain/voorhees/pull/20 that as of this writing is still a pending PR. Considering that it has been open since March 2nd I wouldn't consider that an API that has been accepted or any plans to merge upstream. The PR proposal was my work alone and completely changed how Voorhees would work.

Outside of the idea that both libraries test jsonapi payloads I don't see the case. My tweet indicated that I restarted the effort due to lack of movement on Voorhees. I'm open to debate though.

**Comment by danmcclain** [comment]
Lack of movement in the library was due to lack of documentation and issues with the error throwing of test cases. It seemed like the reason for restarting the library was to brand with DockYard rather than inability to integrate your requests. We had previously discussed the changes of API that were impending, and how the branch your contributions were merged into was pretty much considered the master branch. But fine, if you would not like to attribute where many of the ideas and approaches (like not matching IDs if they were excluded) that's fine

**Comment by bcardarella** [comment]
I don't see any reminder pings on the GitHub thread. It wasn't clear to me if the PR would ever be accepted and the way Voorhees was testing jsonapi payloads was becoming difficult to absorb into our estimations. The composable-style that I came up with is an order of magnitude faster to test with, IMO. We needed a released version of this style for client apps.

FWIW, this work was not a fork in any way of Voorhees. While the idea itself may have been on-track to land in Voorhees at one point the timeline of that became less clear over time. The risk we have to absorb is which libraries we use with our clients. To mitigate that risk I decided to create a brand new library with the compoable concept that I was proposing.
